source/locale/zh_CN/LC_MESSAGES/tutorial/10_key_features/003_efficient_optimization_algorithms.po

```diff
@@ -0,0 +1,333 @@
+"Benchmarks-with-Kurobako>`_."
+msgstr "在大多数例子中我们采用的 :class:`optuna.pruners.MedianPruner`, 尽管其性能基本上会被 "
+":class:`optuna.pruners.SuccessiveHalvingPruner` 和 "
+":class:`optuna.pruners.HyperbandPruner` 超过，就像在 `这个基准测试结果 "
+"<https://github.com/optuna/optuna/wiki/%5BUnder-Construction%5D-"
+"Benchmarks-with-Kurobako>`_ 中那样."
+
+#: ../../source/tutorial/10_key_features/003_efficient_optimization_algorithms.rst:132
+msgid "Activating Pruners"
+msgstr "激活 Pruner"
+
+#: ../../source/tutorial/10_key_features/003_efficient_optimization_algorithms.rst:133
+msgid ""
+"To turn on the pruning feature, you need to call "
+":func:`~optuna.trial.Trial.report` and "
+":func:`~optuna.trial.Trial.should_prune` after each step of the iterative"
+" training. :func:`~optuna.trial.Trial.report` periodically monitors the "
+"intermediate objective values. :func:`~optuna.trial.Trial.should_prune` "
+"decides termination of the trial that does not meet a predefined "
+"condition."
+msgstr "要打开剪枝特性的话，你需要在迭代式训练的每一步后调用 "
+":func:`~optuna.trial.Trial.report` 和 "
+":func:`~optuna.trial.Trial.should_prune`."
+" :func:`~optuna.trial.Trial.report` 定期监控目标函数的中间值. :func:`~optuna.trial.Trial.should_prune` "
+"确定终结那些没有达到预先设定条件的 trial."
+
+#: ../../source/tutorial/10_key_features/003_efficient_optimization_algorithms.rst:137
+msgid ""
+"We would recommend using integration modules for major machine learning "
+"frameworks. Exclusive list is :mod:`optuna.integration` and usecases are "
+"available in  `optuna/examples "
+"<https://github.com/optuna/optuna/tree/master/examples/>`_."
+msgstr "我们推荐在主流机器学习框架中使用集成模块，全部的模块列表在 :mod:`optuna.integration` 里，"
+"用例见 `optuna/examples "
+"<https://github.com/optuna/optuna/tree/master/examples/>`_."
+
+#: ../../source/tutorial/10_key_features/003_efficient_optimization_algorithms.rst:184
+msgid "Set up the median stopping rule as the pruning condition."
+msgstr "将中位数终止规则设置为剪枝条件。"
+
+#: ../../source/tutorial/10_key_features/003_efficient_optimization_algorithms.rst:230
+msgid ""
+"As you can see, several trials were pruned (stopped) before they finished"
+" all of the iterations. The format of message is ``\"Trial <Trial Number>"
+" pruned.\"``."
+msgstr "如你所见，有几个 trial 在其迭代完成之前被剪枝（终止）了。消息格式是 ``\"Trial <Trial Number>"
+" pruned.\"``."
+
+#: ../../source/tutorial/10_key_features/003_efficient_optimization_algorithms.rst:234
+msgid "Which Sampler and Pruner Should be Used?"
+msgstr "应该使用哪个 pruner 呢？"
+
+#: ../../source/tutorial/10_key_features/003_efficient_optimization_algorithms.rst:236
+msgid ""
+"From the benchmark results which are available at `optuna/optuna - wiki "
+"\"Benchmarks with Kurobako\" <https://github.com/optuna/optuna/wiki"
+"/%5BUnder-Construction%5D-Benchmarks>`_, at least for not deep learning "
+"tasks, we would say that"
+msgstr "对于非深度学习来说，根据 `optuna/optuna - wiki "
+"\"Benchmarks with Kurobako\" <https://github.com/optuna/optuna/wiki"
+"/%5BUnder-Construction%5D-Benchmarks>` 里的基准测试结果，我们推荐"
```

Comment:
ditto.
```suggestion
"\"Benchmarks with Kurobako\" <https://github.com/optuna/optuna/wiki"
"/Benchmarks-with-Kurobako>` 里的基准测试结果，我们推荐"
```

Reply:
Also, the link is not rendered as expected:

![image](https://user-images.githubusercontent.com/3255979/108999511-c0d98000-76e5-11eb-8b11-0b2e366e0893.png)